pkgs/by-name/ti/tinysparql/package.nix

```diff
@@ -100,6 +94,7 @@ stdenv.mkDerivation (finalAttrs: {
   mesonFlags =
     [
       "-Ddocs=true"
+      "-Dsystemd_user_services_dir=${placeholder "out"}/share/systemd/user"
```

Comment:
So I've just noticed that that with this change, we lose a symlink from $out/lib/systemd/user to $out/share/systemd/user.  Does that matter?  If it does, changing this to use lib means we get outputs identical to how they were before this change.

Reply:
Okay, I've switched to using the standard path, so we now get both. List of files in each output is now identical before and after this change.